### ISSUE TITLE
PROBLEMA NO ECLIPSE

### ISSUE THREAD

**Issue by ghost** [cami-la/curso-dio-dominando-ides-java#22]
Olá, professora! Muito boa noite, tudo bem?
Comecei o curso recentemente. Ao tentar instalar o Eclipse, e colocar o diretorio da pasta JAVA, o programa não consegue localizar apresentando a seguinte mensagem de ERRO.
Uma outra dúvida, ao tentar compilar o código informado, não consigo também
![image](https://user-images.githubusercontent.com/120606193/207734556-1c4d0ef3-b8e1-4fda-b7fd-a03e8d1130dd.png)

![imagem_2022-12-14_200359129](https://user-images.githubusercontent.com/120606193/207734480-dad4fa9d-d312-4330-ac20-ed1a50725173.png)

**Comment by cami-la** [comment]
Oi, @matheuses22 tudo bem sim e com você? (:

Então, para conseguir executar o programa por linha de comando, você precisa ter o java instalado na sua máquina.
Abre o **cmd** e executa o comando ` java --version`.

Sobre o Eclipse, clica no Botão Download e escolhe a versão 11 ou 17 e faz o download no próprio Eclipse.

**Comment by ghost** [comment]
> 

![image](https://user-images.githubusercontent.com/120606193/208237518-d09c5faa-9835-4f7e-96f5-dbbcb149a568.png)
 Executei o comando e aparece que está instalado

**Comment by ghost** [comment]
Não consegui encontrar a versão 11 ou 17 do Eclipse, teria como me fornecer um link?

**Comment by ghost** [comment]
Aparentemente consegui resolver! Muito obrigado pela ajuda!

**Comment by cami-la** [comment]
Certo, @matheuses22. Que coisa boaaa! Precisando, fico à disposição! (:
